<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="6" column="4">
     <widget class="QPlainTextEdit" name="output_text">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="8" column="1" colspan="2">
     <widget class="QPlainTextEdit" name="plainTextEdit_2">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="plainText">
       <string>Text:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QPlainTextEdit" name="output_path">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QPushButton" name="button">
      <property name="text">
       <string>Find!</string>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QPlainTextEdit" name="output_text_2">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="plainText">
       <string>Searching text:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="4">
     <widget class="QPlainTextEdit" name="output_path_2">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="plainText">
       <string>Searching path:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QPlainTextEdit" name="ready_status">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QPushButton" name="cancel_button">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QPlainTextEdit" name="ready_status_2">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="plainText">
       <string>Status:</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1" colspan="2">
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="plainText">
       <string>Path:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0" rowspan="6" colspan="4">
     <widget class="QPlainTextEdit" name="output_result">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="7" column="3" colspan="2">
     <widget class="QPlainTextEdit" name="input_path"/>
    </item>
    <item row="8" column="3" colspan="2">
     <widget class="QPlainTextEdit" name="input_text"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
